include/z64save.h

```diff
@@ -523,7 +523,7 @@ typedef enum LinkAge {
 #define EVENTCHKINF_4B 0x4B
 #define EVENTCHKINF_4C 0x4C
 #define EVENTCHKINF_4D 0x4D
-#define EVENTCHKINF_4E 0x4E
+#define EVENTCHKINF_CAUGHT_BY_CASTLE_GUARDS 0x4E
```

Comment:
```suggestion
#define EVENTCHKINF_CAUGHT_BY_CASTLE_GUARDS 0x4E // set but unused
```
or something to that effect.